### ISSUE TITLE
Add Alerts from Basic Manual Testing

### ISSUE THREAD

**Issue by bbertucc** [bbertucc/equalify#113]
## What problem is the feature trying to solve?
Automated WCAG 2.1 testing isn't that good (see the @alphagov [accessibility tool audit](https://alphagov.github.io/accessibility-tool-audit/index.html)). Also, as @digitala11y pointed out, developing reports for manual audits is difficult for users who rely on screen readers. 

## The Solution I'd Like
An "Add Altert" could lead to a basic form with all the alert fields:
- Message: A required text area for sanitized text.
- URL: A required text field where a valid URL is required
- Site: A required dropdown field with all the active sites listed.
- Status: A required dropdown field with the options "Active", "Equalified", and "Ignored".

I assume we'll eventually need tags and code snippets, but we can wait until folks request those features since they require a lot of additional work. Tags, for instance, will require us to create a tag management area. 

I also imagine folks will want a way to bulk upload alerts from a CSV, but again that can wait until users start using it and requesting features.

## Alternatives I've Considered
Manually adding things into the MySQL (unsafe).

## Technical Details
To solve this issue, we'll probably need to create a new `/views/add_alert.php` view and `/actions/add_alert.php` action. The view would need to include relevant form inputs, styled using [boostrap markup](https://getbootstrap.com). `/actions/add_alert.php` file would add form data into the `alerts` table, leveraging existing functions or adding a new function in [db.php](https://github.com/bbertucc/equalify/blob/main/models/db.php). All new data would have to be sanitized so that users couldn't add anything dangerous. Once all that is complete, a link to the view would have to be added to [index.php](https://github.com/bbertucc/equalify/blob/main/index.php). The final code should be tested. I am particularly not sure how [process_alerts.php](https://github.com/bbertucc/equalify/blob/main/helpers/process_alerts.php) will deal with manually added alerts. We should also test the code with users like @digitala11y to make sure the feature satisfies their basic needs to report manual testing alerts. Emphasis on "basic". This issue should satisfy basic needs, leading to more advanced tickets later.

**Comment by bbertucc** [comment]
Put together a few screenshots on possible "New Notice" UX. 

<img width="645" alt="Wireframe of a page with form input for a message, status, source URL, Meta: Code Snippet, Meta: More Info Text, and META: HTML Snippets" src="https://github.com/bbertucc/equalify/assets/46652/db80f4e6-d5bd-44bd-aaa7-fd150495258a">

<img width="641" alt="wireframe with the title bulk notice importer that includes a button to upload CSVs of notices" src="https://github.com/bbertucc/equalify/assets/46652/4a3efd19-cf6b-44cc-8cc5-676469b75a59">

<img width="493" alt="Wireframe with the title Page Visualizer with options to select elements from a visualized page" src="https://github.com/bbertucc/equalify/assets/46652/d3a765b8-caa8-4a46-80e1-2c96e97b9ddc">

**Comment by bbertucc** [comment]
Tagging this with #204 - I am hoping we can work manual testing into that release.

**Comment by bbertucc** [comment]
Adding the 1.1 milestone. Since we are moving away from keeping integrations in our core, a standalone installation of Equalify is only useful if we have some manual way of adding issues/notices. This should be the big next step in Equalify development.

**Comment by bbertucc** [comment]
Note: This issue was updated with more streamlined instructions. The goal of this issue is to add the basics of manual testing. We can add more advanced features later.

**Comment by bbertucc** [comment]
Closing this in favor of #228, which speaks more directly to the technical work that needs to be done.
